### PR TITLE
Lesson 6

### PR DESCRIPTION
**Pull request by oiskritsky**
add header, title, content block, link decoration ":active"

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -1,158 +1,179 @@
...
-			</div>
-		</footer>
-	</body>
+  <body>
+    <main class="main">
+      <header class="header">
+        <div class="header__wrapper">
+          <div class="header__logo">
+            <a href="/" class="header__logo-link">
+              <img src="/img/svg/logo.svg" alt="TripHouse" class="header_logo-pic" />
```
Comment (ghost) [inline Triphouse-project/index.html]: Wrong link.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -1,158 +1,179 @@
...
+            <ul class="header__list">
+              <li class="header__item">
+                <a href="#" class="header__link">Stays</a>
+              </li>
+              <li class="header__item">
+                <a href="#" class="header__link">Attractions</a>
+              </li>
+              <li class="header__item">
+                <a href="#">
+                  <img src="/img/svg/Night/Night.svg" alt="night" />
```
Comment (ghost) [inline Triphouse-project/index.html]: Wrong link.

File: Triphouse-project/css/style.css [hunk 3]
```diff
@@ -134,17 +113,89 @@ body {
...
+
+  transition: border-bottom 0.3s;
 }
 
 .main {
-  background: grey;
+  box-sizing: border-box;
+  max-width: 1440px;
+  min-height: 100vh;
+  background: url(/img/intro-bg.jpg) no-repeat;
```
Comment (ghost) [inline Triphouse-project/css/style.css]: Wrong link.

File: Triphouse-project/index.html [hunk 4]
```diff
@@ -1,158 +1,179 @@
 <!DOCTYPE html>
```
Comment (ghost) [inline Triphouse-project/index.html:1]: 2021-04-16_00h18_12
Your page.

File: Triphouse-project/index.html (hunk 1 above)
Comment (oiskritsky) [reply Triphouse-project/index.html]: fixd

File: Triphouse-project/index.html (hunk 2 above)
Comment (oiskritsky) [reply Triphouse-project/index.html]: fixd

File: Triphouse-project/css/style.css (hunk 3 above)
Comment (oiskritsky) [reply Triphouse-project/css/style.css]: fix

File: Triphouse-project/index.html (hunk 4 above)
Comment (oiskritsky) [reply Triphouse-project/index.html:1]: done

**Pull request merged**